pkg/controlplane/storageversionhashdata/data.go

```diff
@@ -77,9 +77,9 @@ var GVRToStorageVersionHash = map[string]string{
 	"networking.k8s.io/v1/ingressclasses":                                  "l/iqIbDgFyQ=",
 	"node.k8s.io/v1/runtimeclasses":                                        "WQTu1GL3T2Q=",
 	"node.k8s.io/v1beta1/runtimeclasses":                                   "WQTu1GL3T2Q=",
-	"policy/v1/poddisruptionbudgets":                                       "6BGBu0kpHtk=",
-	"policy/v1beta1/poddisruptionbudgets":                                  "6BGBu0kpHtk=",
-	"policy/v1beta1/podsecuritypolicies":                                   "khBLobUXkqA=",
+	"policy/v1/poddisruptionbudgets":                                       "EVWiDmWqyJw=",
+	"policy/v1beta1/poddisruptionbudgets":                                  "EVWiDmWqyJw=",
+	"policy/v1beta1/podsecuritypolicies":                                   "bAz9Pn7x8wg=",
```

Comment:
podsecuritypolicies must not change, there's no v1 version to encode to (I would not expect this test to pass)